### DISCUSSION TITLE
Maui Blazor (Desktop, Mobile and web)

### DISCUSSION THREAD

**Discussion by pfaner** [dotnet/maui#4402] [General]
I recently started a new project on Blazor Server. It is a big project, and the development will probably take me all year.
I am pretty sure that I will also need a mobile version because I need to access the device's camera.
I have seen that with Blazor webassembly using JavaScript it's possible to access the camera, however I prefer the advantages of a native application.
There are plans to unify on the same project (Desktop, Mobile and Web)?.
In the future, will it be easier to integrate both projects if I develop it for web assembly?
I have chosen Blazor server mainly because I have a lot of experience with web forms.

Thanks

**Comment by mo3in** [comment]
yes, you can access to camera with blazor (actually in js) in Desktop, Mobile and Web!
with some limitation around ios ...
